University-advisor/Forms/LoginForm.cs

```diff
@@ -39,5 +46,30 @@ private void SignUpButton_Click(object sender, EventArgs e)
             Hide();
             signupForm.Show();
         }
+
+        private bool ValidateFields(string username, string password)
```

Comment:
I think a better option would be to user WHERE clause in sql to find if a user exists instead of fetching all users ant iterating through them.